### PR TITLE
Fix inconsistent alerts by using the markdown notation - part 3

### PR DESCRIPTION
**Pull request by akordowski**
### Why:

As described in the issue #35195 the display of alerts is inconsistent across the documentation.

As the original PR #35216 exceeds the linter limit of 300 files, this PR is part 3 of the changes. It provides changes to alerts across the documentation according to the docs [guidelines](https://docs.github.com/en/contributing/writing-for-github-docs/using-markdown-and-liquid-in-github-docs#alerts) using the Markdown notation.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Alerts for `note`, `tip`, `important` and `warning` have been changed and unified from the liquid notation (`{% note %}`) to the Markdown notation (`> [!NOTE]`) according to the docs guidelines.

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
Just a note for the record! This [series of PRs](https://github.com/github/docs/issues/35195#issuecomment-2462998778) will be reviewed and merged in order, to keep this process as simple and efficient as possible 💛

File: content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ To troubleshoot, make sure that your `_config.yml` file follows these rules:
 
 This error means that one of the pages on your site includes an invalid datetime.
 
-To troubleshoot, search the file in the error message and the file's layouts for calls to any date-related Liquid filters. Make sure that any variables passed into date-related Liquid filters have values in all cases and never pass `nil` or `""`. For more information, see [Filters](https://shopify.dev/docs/api/liquid/filters) in the Liquid documentation.
```
Comment (nguyenalex836) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md:68]: @akordowski Howdy! 👋 It looks like an update made by our team is being reverted here - https://github.com/github/docs/commit/25182f431f6d533fd41ce89c4320a7b530d11a22

Can we ensure the update our team made remains intact? 💛

File: content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md [hunk 2]
```diff
@@ -121,22 +121,22 @@ To troubleshoot, make sure the YAML files in your __data_ folder follow these ru
 
 {% data reusables.pages.yaml-linter %}
 
-For more information about Jekyll data files, see [Data Files](https://jekyllrb.com/docs/datafiles/) in the Jekyll documentation.
```
Comment (nguyenalex836) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md:124]: @akordowski Similar situation here! It looks like these quotation marks were recently removed by our team in an update - https://github.com/github/docs/commit/25182f431f6d533fd41ce89c4320a7b530d11a22

Can we ensure the quotation marks remain absent? 💛

File: content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md [hunk 3]
```diff
@@ -121,22 +121,22 @@ To troubleshoot, make sure the YAML files in your __data_ folder follow these ru
...
+For more information about Jekyll data files, see "[Data Files](https://jekyllrb.com/docs/datafiles/)" in the Jekyll documentation.
 
 ## Markdown errors
 
 This error means that your repository contains Markdown errors.
 
 To troubleshoot, make sure you are using a supported Markdown processor. For more information, see "[AUTOTITLE](/pages/setting-up-a-github-pages-site-with-jekyll/setting-a-markdown-processor-for-your-github-pages-site-using-jekyll)."
 
-Then, make sure the file in the error message uses valid Markdown syntax. For more information, see [Markdown: Syntax](https://daringfireball.net/projects/markdown/syntax) on Daring Fireball.
```
Comment (nguyenalex836) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md:132]: @akordowski Similiarly to comment above, can we ensure the quotation marks remain absent? 💛

File: content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md [hunk 4]
```diff
@@ -121,22 +121,22 @@ To troubleshoot, make sure the YAML files in your __data_ folder follow these ru
...
+Then, make sure the file in the error message uses valid Markdown syntax. For more information, see "[Markdown: Syntax](https://daringfireball.net/projects/markdown/syntax)" on Daring Fireball.
 
 ## Missing docs folder
 
 This error means that you have chosen the `docs` folder on a branch as your publishing source, but there is no `docs` folder in the root of your repository on that branch.
 
 To troubleshoot, if your `docs` folder was accidentally moved, try moving the `docs` folder back to the root of your repository on the branch you chose for your publishing source. If the `docs` folder was accidentally deleted, you can either:
-* Use Git to revert or undo the deletion. For more information, see [git-revert](https://git-scm.com/docs/git-revert.html) in the Git documentation.
```
Comment (nguyenalex836) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md:139]: @akordowski Similarly to comment above, can we ensure the quotation marks remain absent? 💛

File: content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md [hunk 5]
```diff
@@ -146,38 +146,38 @@ This error means that your repository includes a submodule that doesn't exist or
 
 {% data reusables.pages.remove-submodule %}
 
-If you do want to use a submodule, initialize the submodule. For more information, see [Git Tools - Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) in the _Pro Git_ book.
```
Comment (nguyenalex836) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md:149]: @akordowski Similarly to comment above, can we ensure the quotation marks remain absent? 💛

File: content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md [hunk 6]
```diff
@@ -146,38 +146,38 @@ This error means that your repository includes a submodule that doesn't exist or
...
+If you do want to use a submodule, initialize the submodule. For more information, see "[Git Tools - Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules)" in the _Pro Git_ book.
 
 ## Relative permalinks configured
 
 This errors means that you have relative permalinks, which are not supported by {% data variables.product.prodname_pages %}, in your `_config.yml` file.
 
-Permalinks are permanent URLs that reference a particular page on your site. Absolute permalinks begin with the root of the site, while relative permalinks begin with the folder containing the referenced page. {% data variables.product.prodname_pages %} and Jekyll no longer support relative permalinks. For more information about permalinks, see [Permalinks](https://jekyllrb.com/docs/permalinks/) in the Jekyll documentation.
```
Comment (nguyenalex836) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md:155]: @akordowski Similarly to comment above, can we ensure the quotation marks remain absent? 💛

File: content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md [hunk 7]
```diff
@@ -146,38 +146,38 @@ This error means that your repository includes a submodule that doesn't exist or
...
+Permalinks are permanent URLs that reference a particular page on your site. Absolute permalinks begin with the root of the site, while relative permalinks begin with the folder containing the referenced page. {% data variables.product.prodname_pages %} and Jekyll no longer support relative permalinks. For more information about permalinks, see "[Permalinks](https://jekyllrb.com/docs/permalinks/)" in the Jekyll documentation.
 
 To troubleshoot, remove the `relative_permalinks` line from your `_config.yml` file and reformat any relative permalinks in your site with absolute permalinks. For more information, see "[AUTOTITLE](/repositories/working-with-files/managing-files/editing-files)."
 
 ## Syntax error in 'for' loop
 
 This error means that your code includes invalid syntax in a Liquid `for` loop declaration.
 
-To troubleshoot, make sure all `for` loops in the file in the error message have proper syntax. For more information about proper syntax for `for` loops, see [Tags](https://shopify.dev/docs/api/liquid/tags/for) in the Liquid documentation.
```
Comment (nguyenalex836) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md:163]: @akordowski One more update from our team being reverted here! https://github.com/github/docs/commit/25182f431f6d533fd41ce89c4320a7b530d11a22

File: content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md [hunk 8]
```diff
@@ -146,38 +146,38 @@ This error means that your repository includes a submodule that doesn't exist or
...
+To troubleshoot, make sure all `for` loops in the file in the error message have proper syntax. For more information about proper syntax for `for` loops, see "[Iteration tags](https://help.shopify.com/en/themes/liquid/tags/iteration-tags#for)" in the Liquid documentation.
 
 ## Tag not properly closed
 
 This error message means that your code includes a logic tag that is not properly closed. For example, {% raw %}`{% capture example_variable %}` must be closed by `{% endcapture %}`{% endraw %}.
 
-To troubleshoot, make sure all logic tags in the file in the error message are properly closed. For more information, see [Tags](https://shopify.dev/docs/api/liquid/tags) in the Liquid documentation.
```
Comment (nguyenalex836) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md:169]: @akordowski One more update from our team being reverted here! https://github.com/github/docs/commit/25182f431f6d533fd41ce89c4320a7b530d11a22

File: content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md [hunk 9]
```diff
@@ -146,38 +146,38 @@ This error means that your repository includes a submodule that doesn't exist or
...
+To troubleshoot, make sure all logic tags in the file in the error message are properly closed. For more information, see "[Liquid tags](https://help.shopify.com/en/themes/liquid/tags)" in the Liquid documentation.
 
 ## Tag not properly terminated
 
 This error means that your code includes an output tag that is not properly terminated. For example, {% raw %}`{{ page.title }` instead of `{{ page.title }}`{% endraw %}.
 
-To troubleshoot, make sure all output tags in the file in the error message are terminated with `}}`. For more information, see [Objects](https://shopify.dev/docs/api/liquid/objects) in the Liquid documentation.
```
Comment (nguyenalex836) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md:175]: @akordowski One more update from our team being reverted here! https://github.com/github/docs/commit/25182f431f6d533fd41ce89c4320a7b530d11a22

File: content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md [hunk 10]
```diff
@@ -146,38 +146,38 @@ This error means that your repository includes a submodule that doesn't exist or
...
+To troubleshoot, make sure all output tags in the file in the error message are terminated with `}}`. For more information, see "[Liquid objects](https://help.shopify.com/en/themes/liquid/objects)" in the Liquid documentation.
 
 ## Unknown tag error
 
 This error means that your code contains an unrecognized Liquid tag.
 
-To troubleshoot, make sure all Liquid tags in the file in the error message match Jekyll's default variables and there are no typos in the tag names. For a list of default variables, see [Variables](https://jekyllrb.com/docs/variables/) in the Jekyll documentation.
```
Comment (nguyenalex836) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md:181]: @akordowski One more update from our team being reverted here! https://github.com/github/docs/commit/25182f431f6d533fd41ce89c4320a7b530d11a22

**Comment by akordowski** [comment]
@nguyenalex836 I will review the files over the weekend also for the part 4 branch. You can then continue with the review next week. Thank you!

**Comment by akordowski** [comment]
@nguyenalex836 I fixed and reviewed the branches part 3 & 4. If you notice something please notify me. Thank you!

**Review by nguyenalex836 — APPROVED** [review]

**Comment by nguyenalex836** [comment]
@akordowski Part 3 approved! ✨ 🚀 

I do need to grab a review from our Site Policy team since our site policy files were modified, but we should hopefully have this merged soon after 💛 thank you!!

**Review by khxu — APPROVED** [review]

**Pull request merged**